Source/WebKit/WebProcess/WebPage/WebPage.cpp

```diff
@@ -8876,20 +8876,26 @@ bool WebPage::isUsingUISideCompositing() const
 void WebPage::setLinkDecorationFilteringData(Vector<WebCore::LinkDecorationFilteringData>&& strings)
 {
     m_linkDecorationFilteringData.clear();
-    m_domainScopedLinkDecorationFilteringData.clear();
 
-    for (auto& data : strings) {
-        if (data.domain.isEmpty()) {
-            m_linkDecorationFilteringData.add(data.linkDecoration);
+    for (const auto& data : strings) {
```

Comment:
Nit - I believe we generally prefer `auto&` over `const auto&`, even if `data` isn't being mutated.